mirdata/__init__.py

```diff
@@ -6,16 +6,21 @@
 
 from .version import version as __version__
 
-from . import beatles
-from . import ikala
-from . import medleydb_melody
-from . import medleydb_pitch
-from . import medley_solos_db
-from . import orchset
-from . import rwc_classical
-from . import rwc_jazz
-from . import rwc_popular
-from . import salami
-from . import tinysol
-from . import guitarset
-from . import gtzan_genre
+__all__ = [
+    'beatles',
+    'dali',
+    'gtzan_genre',
+    'guitarset',
+    'ikala',
+    'medley_solos_db',
+    'medleydb_melody',
+    'medleydb_pitch',
+    'orchset',
+    'rwc_classical',
+    'rwc_jazz',
+    'rwc_classical',
+    'salami',
+    'tinysol',
+]
+
+from . import *
```

Comment:
presumably, this is only `utils` and `download_utils` and `jams_utils`? if so, it could be nice to include them explicitly so that we don't import modules accidentally.

pyflakes: `mirdata/__init__.py:26: 'from . import *' used; unable to detect undefined names`

Reply:
@rabitt and i discussed about this offline and we think it's better to avoid `from . import *` so that importing mirdata remains fast